lib/collection/src/update_handler.rs

```diff
@@ -506,14 +508,18 @@ impl UpdateHandler {
                 .collect::<Vec<_>>()
                 .into_iter()
                 .rev()
-                .map(|i| handles.remove(i))
+                .map(|i| handles.swap_remove(i))
```

Comment:
Using `swap_removes` means losing ordering of handles.
Just want to confirm that it is not something we care about :)

Reply:
It doesn't matter, but thanks for confirming!

We only keep the handles to count the tasks that are still ongoing, and to catch any panics. Tasks themselves run asynchronously in their own thread. We don't care in what order or at what time we clean them up.